internal/commands/untrust_builder_test.go

```diff
@@ -126,6 +126,30 @@ func testUntrustBuilderCommand(t *testing.T, when spec.G, it spec.S) {
 				)
 			})
 		})
+
+		when("builder is a suggested builder", func() {
+			it("does nothing and reports that ", func() {
+				builder := "gcr.io/paketo-buildpacks/builder:base"
+				cfg := configManager.configWithTrustedBuilders()
+				command := commands.UntrustBuilder(logger, cfg)
+				command.SetArgs([]string{builder})
+
+				h.AssertNil(t, command.Execute())
+
+				b, err := ioutil.ReadFile(configPath)
+				h.AssertNil(t, err)
+				h.AssertNotContains(t, string(b), builder)
```

Comment:
Is this assertion valuable in this context? There's no reason why the suggested builder would appear in this file, especially after all we've invoked is `UntrustBuilder`